service/src/main/java/tennisclub/service/BookingService.java

```diff
@@ -0,0 +1,31 @@
+package tennisclub.service;
+
+import tennisclub.entity.Booking;
+import tennisclub.entity.Court;
+import tennisclub.entity.User;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public interface BookingService {
+
+    void createBooking(Booking booking);
+
+    Booking updateBooking(Booking booking);
+
+    void deleteBooking(Booking booking);
+
+    void removeUser(Booking booking, User user);
+
+    void addUser(Booking booking, User user);
+
+    List<Booking> findALL();
+
+    Booking findById(Long id);
+
+    List<Booking> findByUser(User user);
```

Comment:
I'm not sure if this method makes sense 😆

Reply:
Well, a user might want to view a list of his bookings. So we need to be able to find all bookings of a certain user.